atmo/clusters/forms.py

```diff
@@ -19,14 +20,21 @@ class EMRReleaseChoiceField(forms.ModelChoiceField):
     group for easier selection.
     """
     def __init__(self, *args, **kwargs):
+        emr_releases = models.EMRRelease.objects.active().order_by(
+            RegexpReplace('version', '\\D', '', 'g')
```

Comment:
This basically replaces all non-digit characters with an empty string and sorts by it.